x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/indexlifecycle/action/TransportMoveToStepAction.java

```diff
@@ -56,6 +60,8 @@ protected void masterOperation(Request request, ClusterState state, ActionListen
             new AckedClusterStateUpdateTask<Response>(request, listener) {
                 @Override
                 public ClusterState execute(ClusterState currentState) {
+                    logger.info("moving index [{}] from [{}] to [{}]",
```

Comment:
Would it be possible for us to pull the policy name out here too and put it in? I think that might be nice

Reply:
In order to do this I needed to move the log statement to `IndexLifecycleRuner#moveClusterStateToStep` as we don't know the policy in the transport action. However, the javadoc for that method says that it's intended for moving steps in response to `TransportAction` executions, so I think it's fine to be there instead. That also lets us emit the logs after other checks (e.g. making sure the index is actually managed by any policy at all).

Reply:
That's going to log a lot more often, since that's called for almost every step move, is that what we want?

Reply:
`IndexLifecycleRuner#moveClusterStateToStep` is only called via `TransportMoveToStepAction` and `TransportRetryAction`, I think you're thinking of `IndexLifecycleRunner#moveClusterStateToNextStep`.

Reply:
These method names could use a bit of work.

Reply:
Yes, they definitely could, okay, I'm good with this where it is then :)